### DISCUSSION TITLE
libzippp and libzip versioning

### DISCUSSION THREAD

**Discussion by SvenKnabe** [microsoft/vcpkg#34997] [Q&A] [answered]
Hello together,
I use vcpkg in my project, but haven't that much experience with it. 

I use the library libzippp which is a C++ wrapper for the library libzip. libzippp is available with version 7.0-1.10.1 under vcpkg (so indicates that the wrapper is updated to support libzip version 1.10.1). But libzip itself is only available under version 1.9.2 under vcpkg.  

Q1: Does this now mean, that I only get an updated wrapper but the libzip stays with version 1.9.2?

In my project I need at least libzip version 1.10.0 because this introduces OpenSSL 3 support. Assuming, Q1 is correct, do I have to open a new issue to request updating libzip to version 1.10.1?

Thanks to anyone who can help me here.

**Comment by StarGate-One** [comment, marked as answer]
All you need to do is open an issue using the following template: [request a port update](https://github.com/microsoft/vcpkg/issues/new?assignees=&labels=category%3Aport-update&projects=&template=02-request-an-update-to-an-existing-port.yml&title=%5B%3Cport+name%3E%5D+update+to+%3Cversion%3E)

**Reply by SvenKnabe** [reply]
Thank you, I did.

**Reply by StarGate-One** [reply]
You are most welcome!
